### DISCUSSION TITLE
Issues regarding WT4 and EV1

### DISCUSSION THREAD

**Discussion by sghqsads** [CURENT/andes#515] [Q&A]
Dear Andes developers,

I connect WT4 (REPCA1, REECA1, REGCA1) and EV1 in a same bus in Andes. However, I cannot get the corresponding WT4 output by setting Plant_pref (EV1 output has already been deducted). I would like to inquire whether this is due to WT4 and EV1 cannot be utilized in a same bus in Andes or there maybe other precautions.

Thank you!

**Comment by jinningwang** [comment]
Ensure the plant control is connected to the corresponding wind model.
